tpm2/credactivation/credential_activation.go

```diff
@@ -47,37 +50,28 @@ const (
 // by the TPM. A 32 byte secret is a safe, recommended default.
 //
 // This function implements Credential Protection as defined in section 24 of the TPM
-// specification revision 2 part 1, with the additional caveat of not supporting ECC EKs.
+// specification revision 2 part 1.
```

Comment:
nit: prefer "TPM 2.0 specification" over "TPM specification revision 2" (since the 2.0 specification has 'revisions')